src/cascadia/TerminalSettingsModel/CascadiaSettings.idl

```diff
@@ -57,6 +59,7 @@ namespace Microsoft.Terminal.Settings.Model
 
         String AIEndpoint;
         String AIKey;
+        static event AzureOpenAISettingChangedHandler AzureOpenAISettingChanged;

```

Comment:
This is going to be moved into the new `AIConfig` struct in the follow up PR that introduces OpenAI

Reply:
A static _event_? that's weird to me but okay. It's moving soon so I'll look at it there

Reply:
> A static event
Yep, that's because these strings aren't actually saved in our settings, they're saved in the credential manager. Which means that if there's multiple instances of terminal open (each with their own TerminalChat), and the auth is changed in one of them, this is what we have to emit to get all instances to update.